wgpu-types/src/lib.rs

```diff
@@ -828,6 +962,48 @@ pub struct DepthStencilStateDescriptor {
 }
 
 impl DepthStencilStateDescriptor {
+    pub fn new(format: TextureFormat) -> Self {
+        Self {
+            format,
+            depth_write_enabled: false,
+            depth_compare: CompareFunction::Undefined,
+            stencil_front: StencilStateFaceDescriptor::default(),
+            stencil_back: StencilStateFaceDescriptor::default(),
+            stencil_read_mask: 0,
+            stencil_write_mask: 0,
+        }
+    }
+
+    pub fn depth_write_enabled(&mut self, enable: bool) -> &mut Self {
```

Comment:
let's remove the parameter
I think we should rename the method to just `depth_write`, as other booleans don't have "_enabled" suffix

Reply:
Should `depth_write_enabled` and `depth_compare` be set together? And also, should `depth_compare` be initialized with `CompareFunction::Undefined`?